### DISCUSSION TITLE
[Kubernetes] CKAD/CKA 證照考試心得 | 全端開發人員天梯

### DISCUSSION THREAD

**Discussion by giscus** [wellwind/wellwind.github.io#86] [Announcements]
# [Kubernetes] CKAD/CKA 證照考試心得 | 全端開發人員天梯

為了即將到來的工作需求，最近開始準備 Kubernetes 的考試，花了一個月的時間學習，總算取得 CKAD 和 CKA 證照，在這邊分享一下我這一個月的學習心得。

https://fullstackladder.dev/blog/2023/07/29/how-to-pass-kubernetes-ckad-cka-certificate/

**Comment by youling4438** [comment]
恭喜恭喜.

**Comment by MickeyShiue** [comment]
CKA 考試簡介底下的
在撰寫這篇文章的時間點，考試內容範圍包含以下，詳細內容可以到 CKAD 考試網站 查看。
這邊好像放錯連結了XD

**Reply by wellwind** [reply]
非常感謝，更正囉 XD
